docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,374 @@
+- Learn how to instantiate the UID2Manager instance in your Android app: see [Usage Guidelines](../sdks/uid2-sdk-ref-android.md#usage-guidelines).
+
+(**GWH__SW_35 is the Android link correct? It was a link to "Android Initialization" which doesn't exist. Showed up on a broken links check and I updated it as above.**)
+ 
+ For iOS, refer to the [Usage Guidelines](../sdks/uid2-sdk-ref-ios.md#usage-guidelines).
+
+## Using the UID2 Integration Environment
+
+By default, the SDK is configured to work with the UID2 production environment: `https://prod.uidapi.com`. If you want to use the UID2 integration environment instead, provide the following URL in your call to UID2Manager initialization:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://operator-integ.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+Bear in mind the following differences between environments:
+- Tokens from the UID2 integration environment are not valid for passing to the bid stream.
+- You'll have a different set of Subscription ID and public key values for each environment (integ and prod). Be sure to use the correct values for each environment.
+:::
+
+## Optional: Reduce Latency by Setting the API Base URL for the Production Environment
+
+<ReduceLatency />
+
+To specify a different UID2 server, you can change it in the `init` call:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = " https://global.prod.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Settings.shared.environment = .singapore
+// or
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://global.prod.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+## Configure the UID2 Mobile SDK
+
+After you've instantiated `UID2Manager` correctly in your mobile app, you'll need to pass a UID2 <Link href="../ref-info/glossary-uid#gl-identity">identity</Link> generated server-side (see [Implement server-side token generation](#implement-server-side-token-generation)), and then pass it into the mobile app using the `setIdentity` method, as shown in the following:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().setIdentity()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.setIdentity()
+```
+
+</TabItem>
+</Tabs>
+
+(**GWH_SW_32 line 251 above you said: "should be setIdentity() instead" but I'm not sure of the whole of it. I don't see the anywhere else not sure it's correct. https://github.com/IABTechLab/uid2docs/pull/557/files#r1598149873**)
```

Comment:
looks correct now - maybe the japanese version was wrong and i left the comment there...

Reply:
@sunnywu to be clear... I did update this line of code but just updated the function name at the end of it, not the first part of the line.
But you say it looks fine now so should be good.

Reply:
All good per discussion thx.